templates/site/sample.html

```diff
@@ -0,0 +1,302 @@
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    <br>
+    <br>
+
+
+    {% with %}
+        {% set button_type = "button_lg_nav_previous" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_med_select_ascending" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_med_select_descending" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_lg_nav_next" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_sm_nav" %}
+        {% set button_label = "First" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_sm_default_open" %}
+        {% set button_label = "Open" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_sm_default_copy" %}
+        {% set button_label = "Copy" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_reg_default" %}
+        {% set button_label = "Apply" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    {% with %}
+        {% set button_type = "button_reg_default-alt" %}
+        {% set button_label = "Clear" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+
+
+    {% with %}
+        {% set button_type = "close_button_x" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    <br>
+    <br>
+
+    {% with %}
+        {% set button_type = "radio_button_sm" %}
+        {% set radio_button_id = "radioButtonSmall" %}
+        {% set radio_button_label = "" %}
+        {% set radio_button_orientation_class = "" %}
+        {% include "partials/button.html" %}
+    {% endwith %}
+
+    <br>
+    <br>
+
+    {% with %}
+        {% set datepicker_id = "inputDateReg" %}
+        {% set datepicker_label = "Date:" %}
+        {% include "partials/datepicker.html" %}
+    {% endwith %}
+
+        <br>
+    <br>
+
+    {% with %}
+        {% set timepicker_id = "inputTimeReg" %}
+        {% set timepicker_label = "Time:" %}
+        {% include "partials/timepicker.html" %}
+    {% endwith %}
+
+    <br>
+    <br>
+
+
+    {% with %}
+        {% include "partials/tag.html" %}
+    {% endwith %}
+
+    <br>
+    <br>
+
+    {% with %}
+        {% set display_type = "display_row" %}
+        {% set display_link = "tasmin_day_UKESM1-0-LL_ssp585_r1i1p1f2_gn_2100" %}
+        {% set start_date = "1/1/2100" %}
+        {% set end_date  = "12/30/2100" %}
+        {% set start_time = "12:0:0 PM" %}
+        {% set end_time = "12:0:0 PM" %}
+        {% include "partials/info-display.html" %}
+    {% endwith %}
+
+        <br>
+    <br>
+
+    {% with %}
```

Comment:
This is part of a grid. Please create the grid component as well and show it with multiple cards